plusGitRepo/main/index.js

```diff
@@ -111,14 +139,27 @@ define([
                   .text("Clone")
                             .on("click", function(e) {
                               var plusGitRepoUrl =
-                                  $( "input#plusGitRepoUrl" ).val();
+                                $( "input#plusGitRepoUrl" ).val();
                               var plusGitRepoBranch =
-                              $( "input#plusGitRepoBranch" ).val();
+                                $( "input#plusGitRepoBranch" ).val();
+                              var plusGitRepoUsername = 
+                                $( "input#plusGitRepoUsername" ).val();
+                              var plusGitRepoPassword =
+                                $( "input#plusGitRepoPassword" ).val();
                               if (plusGitRepoUrl && plusGitRepoBranch ) {
                                 if (plusGitRepoUrl.indexOf("http") === 0)
                                 {
                                   // construct the url that nbgitpuller needs
                                   var thisUrl = window.location.href;
+                                  console.log("plusGitRepoUsername:" + plusGitRepoUsername + " && plusGitRepoPassword:" + plusGitRepoPassword)
```

Comment:
Could you remove the logging statements here and on line 162?

Reply:
Done....